docker/pypi/dmwm-base/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM registry.cern.ch/cmsweb/cmsweb-base as cmsweb-base
-FROM registry.cern.ch/cmsweb/exporters as exporters
-FROM python:3.8-bullseye
+FROM registry.cern.ch/cmsweb/cmsweb-base:20230901-stable as cmsweb-base
+FROM registry.cern.ch/cmsweb/exporters:20231002-stable as exporters
```

Comment:
same here, we should rebuild stable image for exporters and use the newest one, i.e. 202505xx-stable

Reply:
i opened a tentative PR https://github.com/dmwm/CMSKubernetes/pull/1613

Reply:
meged